app/assets/stylesheets/components/page_specific/home_beta/_tool_promo.scss

```diff
@@ -0,0 +1,44 @@
+.tool_promo-beta {
+  @include column(12);
+  border: 0;
+
+  @include respond-to($mq-m) {
+    @include column(6);
+  }
+
+  @extend .callout;
+  @extend .callout--tool;
+}
+
+.tool_promo-beta__item {
+  @include body(14, 24);
+  background-color: $color-grey-ten;
+  border-bottom: 2px solid $color-orange-medium;
+  padding: $baseline-unit*2;
+  color: black;
+  position: relative;
+}
+
+a.tool_promo-beta__item-link {
+  color: $color-grey-primary;
+
+  @include respond-to($mq-m) {
+    padding: $baseline-unit*2 $baseline-unit*3.5 $baseline-unit*2;
+    display: block;
+    text-align: right;
+  }
+}
+
+.tool_promo-beta__item-heading {
+  color: $color-grey-primary;
+  margin: 0;
+}
+
+.tool_promo-beta__item-arrow {
+  fill: $color-orange-medium;
+  height: 34px;
+  position: absolute;
+  right: 2%;
+  top: 23px;
+  width: 30px;
+}
```

Comment:
This element on the final row is close to the bottom of the component. May be better if all these are vertically centred. 

![image](https://user-images.githubusercontent.com/6080548/34872916-cce937ce-f78a-11e7-85d3-7a2adb8879ae.png)